internal/helper/json.go

```diff
@@ -0,0 +1,21 @@
+package helper
+
+import "encoding/json"
+
+// ToJSON returns the JSON string of v or panics.
+func ToJSON(v any) string {
```

Comment:
Should this be called 

```suggestion
func MustToJSON(v any) string {
```

I'm still not sure we should be panicing in a lib like this. The caller might have a strategy for handling failure. For example, Kraken is down or broken and returning incorrect JSON so they fallback to increasing their hedge position on some other market.